gradio-app/gradio
# Ensure `gradio_client` is backwards compatible with `gradio==3.24.1`

[As discussed internally](https://huggingface.slack.com/archives/C02QZLG8GMN/p1690856034533989), some of the changes to the client were not fully backwards compatible, which broke older versions of `gradio` if they were using later versions of the client. This fixes that, at least for `gradio==3.24.1` (I have not tested other versions of `gradio`). 

To test:

1. Install `gradio==3.24.1`
2. Install the latest version of `gradio_client` from pypi
3. Run any Gradio app (it will not work)
4. Install `gradio_client` from this branch
5. It should work
